nixos/modules/services/networking/go-camo.nix

```diff
@@ -0,0 +1,62 @@
+{ lib, pkgs, config, ... }:
+with lib;
+
+let
+  cfg = config.services.go-camo;
+in
+{
+  options.services.go-camo = {
+    enable = mkEnableOption "go-camo service";
+    addressPort = mkOption {
```

Comment:
Maybe you should just call this option `listen` to remain consistent with upstream.